config/config.go

```diff
@@ -679,6 +680,7 @@ type EC2SDConfig struct {
 	SecretKey       string   `yaml:"secret_key,omitempty"`
 	RefreshInterval Duration `yaml:"refresh_interval,omitempty"`
 	Port            int      `yaml:"port"`
+	SubnetSeparator string   `yaml:"subnet_separator,omitempty"`
```

Comment:
As the list items are IDs I think we can just have a fixed separator for them (comma) rather than making it configurable. For Consul tags it was necessary because we have no guarantees about used characters.